panther_analysis_tool/main.py

```diff
@@ -872,26 +1002,49 @@ def setup_parser() -> argparse.ArgumentParser:
     test_parser.add_argument(ignore_extra_keys_name, **ignore_extra_keys_arg)
     test_parser.set_defaults(func=test_analysis)
 
-    zip_schemas_parser = subparsers.add_parser(
-        "zip-schemas", help="Create a release asset archive of managed schemas."
+    publish_parser = subparsers.add_parser(
+        "publish",
+        help="Publishes a new release, generates the release assets, and uploads them"
+        + "Generates a file called panther-analysis-all.zip and optionally generates "
```

Comment:
In order to not sign it, you have to explicitly set `--kms-key` to an empty string right? So `panther_analysis_tool publish --kms-key ''` or something? Because otherwise I thin it will take the default value of the panther key and try (and presumably fail) to sign it with that. So perhaps we should not set a default value, or maybe make an explicit `--no-sign` flag? Not sure what the best course is but seems strange to need `--kms-key ''` to disable signing.

Reply:
Yea, I was trying to make it as easy as possible, but I agree with you - this seems awkward.  I think I vote for just removing the default value.

Reply:
Sounds good, I'm sure we'll have all these values recorder somewhere for easy access anyways.